clusterloader2/pkg/prometheus/manifests/default/master-serviceMonitor.yaml

```diff
@@ -1,19 +1,25 @@
+{{$PROMETHEUS_SCRAPE_ETCD := DefaultParam .PROMETHEUS_SCRAPE_ETCD false}}
```

Comment:
Do we really gate it?
I think it's useful enough that I think we should simply switch it on...

Reply:
Discussed personally, we are gating it to avoid long process of cherrypicking.

Reply:
I have those "if"s, but ok - let's make it.